homework3.py

```diff
@@ -0,0 +1,71 @@
+# Найти самое длинное слово в введенном предложении.
+# Учтите что в предложении есть знаки препинания
+phrase = input("Please enter some phrase with signs separated by space: ")
+
+print("Entered phrase: ", phrase)
+
+# out = "".join(k for k in phrase if k not in('!', '.', ','))
+# print(out)
+
+signs = ['.', ',', ':', ';', '!', '?', '(', ')',
+         ' .', ' ,', ' :', ' ;', ' !', ' ?', ' (', ' )',
+         '. ', ', ', ': ', '; ', '! ', '? ', '( ', ') ']
+list_word = phrase.split()
```

Comment:
it seems it would be enough to split it and use strip for number of special characters and space (we don't need all these combination signs and spaces).